### PR TITLE
Automatically install Symflower with a fixed version for the CI, development environment and evaluation benchmark

### PR DESCRIPTION
**Pull request by zimmski**

### CODE REVIEW

File: tools/symflower_test.go [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+	t.Run("Install tools a second time which should install no new tools", func(t *testing.T) {
+		logOutput, log := log.Buffer()
+		require.NoError(t, SymflowerInstall(log, temporaryPath))
+
+		require.NotContains(t, logOutput.String(), `Install "symflower" to`)
+		symflowerPath, err := exec.LookPath("symflower")
+		require.NoError(t, err)
+		require.NotEmpty(t, symflowerPath)
+	})
```
Comment (bauersimon) [inline tools/symflower_test.go:45]: Would only use `require` for the errors that are not our fault (i.e. `exec.LookPath`). And then use `assert` for the other assertions.

Also for the integration tests of the command.

File: cmd/eval-dev-quality/cmd/evaluate.go [hunk 2]
```diff
@@ -19,10 +19,14 @@ import (
 	"github.com/symflower/eval-dev-quality/provider"
 	_ "github.com/symflower/eval-dev-quality/provider/openrouter"
 	_ "github.com/symflower/eval-dev-quality/provider/symflower"
+	"github.com/symflower/eval-dev-quality/tools"
 )
 
 // Evaluate holds the "evaluation" command.
 type Evaluate struct {
+	// InstallToolsPath determines where tools for the evaluation are installed.
+	InstallToolsPath string `long:"install-tools-path" description:"Install tools for the evaluation into this path."`
```
Comment (bauersimon) [inline cmd/eval-dev-quality/cmd/evaluate.go:28]: Would be cool to have the default location here for the user as well.

(also for the separate command)

File: tools/symflower.go [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+		}
+		symflowerVersionWanted, err := strconv.ParseUint(m[1], 10, 64)
+		if err != nil {
+			return pkgerrors.WithStack(err)
+		}
+
+		// Binary is installed in a compatible version.
+		if symflowerVersionInstalled >= symflowerVersionWanted {
+			return nil
+		}
```
[the remark this answers is not in the document]
Comment (zimmski) [reply tools/symflower.go:74]: WUPS!

[the remark this answers is not in the document]
Comment (zimmski) [reply tools/symflower.go:74]: Now you know which test i left out ;-(

File: tools/symflower_test.go (hunk 1 above)
Comment (zimmski) [reply tools/symflower_test.go:45]: Don't agree. Require is to stop a test and assert allows to execute further.  Use require if further execution does not make sense.

File: cmd/eval-dev-quality/cmd/evaluate.go (hunk 2 above)
Comment (zimmski) [reply cmd/eval-dev-quality/cmd/evaluate.go:28]: You mean as the "default value"? We cannot change the description dynamically. Code attributes do not allow that.

File: tools/symflower_test.go
[the remark this answers is not in the document]
Comment (zimmski) [reply tools/symflower_test.go:1]: Don't think so. Pretty basic for now.

File: cmd/eval-dev-quality/cmd/evaluate.go (hunk 2 above)
Comment (bauersimon) [reply cmd/eval-dev-quality/cmd/evaluate.go:28]: Yeah I know :crying_cat_face:

**Review by bauersimon — APPROVED** [review]

**Pull request merged**